components/collector/src/source_collectors/sarif/base.py

```diff
@@ -0,0 +1,47 @@
+"""SARIF JSON collector."""
+
+from typing import cast
+
+from base_collectors import JSONFileSourceCollector
+from collector_utilities.type import JSON, JSONDict
+from model import Entities, Entity
+
+
+class SARIFJSONBase(JSONFileSourceCollector):
+    """SARIF collector for security warnings."""
```

Comment:
No longer just security warnings, right?

```suggestion
    """Base class for collectors that read SARIF JSON files."""
```